hkmc2/shared/src/test/mlscript/bbml/bbBasics.mls

```diff
@@ -1,5 +1,5 @@
 :bbml
-//│ Type: ⊤
+//│ Type: ⊥
```

Comment:
I just realized that for some reason the bbml test cases were not resolved by the resolver. I fixed it by modifying `hkmc2DiffTests/src/test/scala/hkmc2/MLsDiffMaker.scala`. 

IDK why this funny T symbol (and in other bbml files) became upside down 🤦 I guess this is related to the automatically inserted import statement. Just take a note and I will take a look at it later.

Reply:
Lol ⊤ is top and ⊥ is bottom. The fact top became bottom is somewhat concerning (an understatement).

Reply:
In BbML, ⊤ is used for "unit type", basically.

Reply:
It's funny that the problem was I forgot that I changed `bbPrelude` 🤡